Dockerfile

```diff
@@ -25,6 +25,7 @@ WORKDIR /
 COPY --from=builder /workspace/manager .
 ENV OPERATOR=/usr/local/bin/search-operator \
     USER_UID=1001 \
-    USER_NAME=search-operator
+    USER_NAME=search-operator \
+    DEPLOY_REDISGRAPH="false"
```

Comment:
Until we update the canaries, we could default to "true" and merge this so we have an easy way to remove RG while we work on the remaining changes.